pkg/api/types.go

```diff
@@ -1249,128 +1249,129 @@ type ContainerTestConfiguration struct {
 type ClusterProfile string
 
 const (
-	ClusterProfileAWS                   ClusterProfile = "aws"
```

Comment:
What happened here so we have so many deletions and additions?

Reply:
It is because of the spacing/alignment. Due to the length of the name:

ClusterProfileRedHatOpenShiftEcosystem ClusterProfile = "redhat-openshift-ecosystem"

The other ClusterProfile above are pushed to the right to align.

Reply:
I would suggest a shorter name, using shortcut then. It would be safer for us (TP) to know only several lines were changed and what PRs introduced what cluster profiles in the past from blame. Realigning that now will change this situation.

Reply:
@jmguzik fixed

Reply:
The linting in this project even enforces this change, and is failing bc of the rollback.

Reply:
Wrong push fixed